.github/workflows/ci-workflow.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, macos-latest, ubuntu-latest]
-        node-version: ["12", "13", "14", "15"]
+        node-version: ["12", "14", "15", "16"]
```

Comment:
How about 12, 14, 16, 18?

Reply:
!Great suggestion

Reply:
There is no version 18, 16 is the latest one that got released in April 2021